products/extensions/b2b-components/employee-management/concept/entities-and-schema.md

```diff
@@ -0,0 +1,47 @@
+# Entities and schema
```

Comment:
I would advice against describing every entity, since the ER diagram already covers the relationships. We should only go into detail if there are crucial details to an entity, you definitely don't get from the diagram.

The overall description of the most important entities (like company, employee, role) is already covered by the introduction page.

Reply:
What do you think @sushmangupta

Reply:
@marcelbrode / @bschulzebaek , Can you point me to the introduction page ?

Reply:
done